File: lattice.core/Throwables/RuntimeException.vb
Namespace Throwables
    Public Class RuntimeException
        Inherits LatticeException
        Public Sub New(message As String, Optional errorCode As String = "R001", Optional helpText As String = "", Optional location As String = "", Optional notes As IEnumerable(Of String) = Nothing)
            MyBase.New(message, errorCode, helpText, location, notes)
        End Sub
    End Class
End Namespace

File: Lattice.Runtime/Runtime/RuntimeException.cs
namespace lattice.Throwables;

public class RuntimeException : LatticeException
{
    public RuntimeException(string message, string errorCode = "R001", string helpText = "", string location = "", IEnumerable<string>? notes = null)
        : base(message, errorCode, helpText, location, notes)
    {
    }
}
